recipes/sherlockml-dataclean/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "sherlockml-dataclean" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "1c3a8b1fe7f83ed796e0c2e3666fc51dd83dc9213fa2c49e3efe26cec2211653" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - jupyter
+  run:
+    - python
+    - pandas
+    - numpy
+    - matplotlib
+    - ipywidgets >=7.0.0
+    - scipy
+    - funcsigs  # [py2k]
+    - scikit-learn
+    - ipython
+    - future
+    - sherlockml-boltzmannclean
+
+test:
+  imports:
+    - dataclean
+
+about:
+  home: https://sherlockml.com
+  license: Apache-2.0
+  license_family: Apache
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: 'Interactive cleaning for pandas DataFrames'
+  description: |
+    A Jupyter plugin for interactive cleaning of pandas DataFrames.
+
+extra:
+  recipe-maintainers:
+    - shwinnn
+    - acroz
+    - neb42
+    - zblz
+    - srstevenson
+    - pbugnion
```

Comment:
@acroz @neb42 @zblz @srstevenson @pbugnion are you willing to be maintainers?

Reply:
Yep.